sycl/include/CL/sycl/platform.hpp

```diff
@@ -102,7 +102,18 @@ class __SYCL_EXPORT platform {
   /// \return a vector of all available SYCL platforms.
   static vector_class<platform> get_platforms();
 
+  /// Gets the native handle of the SYCL platform.
+  ///
+  /// \return a native handle, the type of which defined by the backend.
```

Comment:
```suggestion
  /// \return a native handle, the type of which is defined by the backend.
```

Reply:
I will include the change in my other PR in-flight to not have to re-test this one.